defaults.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
     'apostrophe-images-widgets': {},
     'apostrophe-files': {},
     'apostrophe-files-widgets': {},
-    'apostrophe-service-bridge': {}
+    'apostrophe-service-bridge': {},
+    'apostrophe-notifications': {}
```

Comment:
apostrophe-service-bridge is intended to be last. It lets all the core modules know that all the core modules are ready.